conjur/init/init_controller.py

```diff
@@ -36,11 +41,13 @@ def load(self):
         """
         Method that facilitates all method calls in this class
         """
-        fetched_certificate = self.get_server_certificate()
-        self.write_certificate(fetched_certificate)
+        if self.ssl_verify is True:
+            fetched_certificate = self.get_server_certificate()
+            self.write_certificate(fetched_certificate)
+        else:
+            self.conjurrc_data.cert_file=""
```

Comment:
why we need to do this?

Reply:
because it is set to None at the argparse level because the user did not provide the certificate so we want to purposefully set it to "" so that the entry is still printed because we have in init_logic.py 
```
            # Ensures that there are no None fields written to conjurrc
            for attr,value in conjurrc_data.__dict__.items():
                if value is not None:
                    _pretty_print_object[str(attr)]=value
```
to ensure that the cert file is still being printed but now that i think about it I can remove this check because all values are filled by our work in the CLI

Reply:
Cant we set it to "" in argparse level?

Reply:
ive updated it always